### PR TITLE
Recipe app

### PR DESCRIPTION
**Pull request by codedit334**
# Hey Reviewer 👋,

This project is made by @codedit334 and  @Gmanlove 

Tasks done in this project:
- Group work:
  - #1 
- @codedit334 's work:  
  - #2 
  - #4 
  - #6 
  - #9 
  - #7
- @Gmanlove 's work:
  - #3 
  - #5 
  - #8 
  - #11

# ✋

### CODE REVIEW

File: app/views/general_shopping_lists/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <% @missing_foods.each do |missing_food| %>
+      <tr>
+        <td><%= missing_food[:name] %></td>
+        <td><%= @general_food_list.find_by(name: missing_food[:name])&.measurement_unit %></td>
+        <td><%= @general_food_list.find_by(name: missing_food[:name])&.price %></td>
+        <td><%= missing_food[:quantity] %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```
Comment (cynthiainga) [inline app/views/general_shopping_lists/index.html.erb]: - [ ] As mentioned in the project requirement you should match the given wireframe, kindly make sure this page much the provided design 🙃 

Your design

Screenshot 2023-06-29 at 20 39 08

Given wireframe

Screenshot 2023-06-29 at 20 39 40

- [optional] Also I would suggest you to the `Hello #username` before the navigation bar 😁

File: README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ <ul>
+    <li><a href="https://www.microverse.org/">Microverse</a></li>
+  </ul>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+  This project is [MIT](./LICENSE.txt) licensed.
```
Comment (cynthiainga) [inline README.md]: - [ ] Please kindly fix your MIT link license, it is direct to the page not found 🙃

File: README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** Blog App is a web application that allows users to create, read, update, and delete blog posts. Users can sign up for an account, create new blog posts, and edit or delete their existing posts.
```
Comment (cynthiainga) [inline README.md]: - [ ] Please kindly update the description of your project and the title as well 🙏🏾

File: README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add new post]**
+- **[Add new comment]**
+- **[Add new like]**
+
```
Comment (cynthiainga) [inline README.md]: - [ ] Kindly update your `Key Feature` as well, this project is about Recipe and food not post comment or like 😁

File: README.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+### Preriquisites
+
+Ruby version >= 3.0.0
+
+### Setup
+
+Clone this repository to your desired folder:
+sh
+  cd my-folder
+ git clone https://github.com/ernestmusong/Blog-app.git
```
Comment (cynthiainga) [inline README.md]: - [ ] Kindly add the correct link to clone 🙃

File: README.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Emmanuel Obi**
+- GitHub: [@Gmanlove](https://github.com/Gmanlove)
+
```
Comment (cynthiainga) [inline README.md]: - [OPTIONAL] It would be better if the other author add more contact information like LinkedIn and twitter 😁

File: README.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+- Twitter: [@MusongAkeh](https://twitter.com/MusongAkeh)
+- LinkedIn: [Musong Ernest](https://www.linkedin.com/in/musongernestakeh/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[reply to a comment]**
+- [ ] **[Search for specific blog posts based on keywords or tags]**
+
```
Comment (cynthiainga) [inline README.md]: - [ ] Please kindly update your Future Features to correspond the project 🙃

File: README.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+- [ ] **[reply to a comment]**
+- [ ] **[Search for specific blog posts based on keywords or tags]**
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ernestmusong/Blog-app/issues).
```
Comment (cynthiainga) [inline README.md]: - [ ] When I click to the issue link it direct me to another project `Blog-app` please kindly fix that 🙏🏾

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @codedit334 & @Gmanlove, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Follow the GitHub flow and Gitflow ✔️ 
- [ ] The app work as expected ✔️ 
- [ ] Associations are setup correctly ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @codedit334 and @Gmanlove 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Good App design 💪🏿
- All changes requested have been implemented 🥇  


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codedit334** [comment]
Thanks @prolajumokeoni !

**Comment by Gmanlove** [comment]
Thanks for your  time

**Pull request merged**